Student/src/main/java/com/fnp/model/StudentModel.java

```diff
@@ -14,54 +14,79 @@ public class StudentModel {
 	private String email;
 	@NotNull
 	private long phone;
+
 	/**
 	 * @return the id
 	 */
 	public int getId() {
 		return id;
 	}
+
 	/**
 	 * @param id the id to set
 	 */
 	public void setId(int id) {
 		this.id = id;
 	}
+
 	/**
 	 * @return the name
 	 */
 	public String getName() {
 		return name;
 	}
+
 	/**
 	 * @param name the name to set
 	 */
 	public void setName(String name) {
 		this.name = name;
 	}
+
 	/**
 	 * @return the email
 	 */
 	public String getEmail() {
 		return email;
 	}
+
 	/**
 	 * @param email the email to set
 	 */
 	public void setEmail(String email) {
 		this.email = email;
 	}
+
 	/**
 	 * @return the phone
 	 */
 	public long getPhone() {
 		return phone;
 	}
+
 	/**
 	 * @param phone the phone to set
 	 */
 	public void setPhone(long phone) {
 		this.phone = phone;
 	}
+	
+	
+	
+	public StudentModel() {
+		super();
```

Comment:
is it required ?

Reply:
yes, for creation of object for testing purpose.